src/main/javascript/trends.js

```diff
@@ -577,6 +580,36 @@ class TrendsPage extends React.Component {
         this.displayPlots(airframe);
     }
 
+
+    getDescription(eventName) {
+        var text = "";
+
+        $.ajax({
+                type: 'GET',
+                url: '/protected/get_event_description',
+                data : {eventName : eventName},
+                dataType : 'json',
+                success : function(response) {
+                    console.log("received response: " + response);
+
+                    $('#loading').hide();
+
+                    if (response.err_msg) {
+                        errorModal.show(response.err_title, response.err_msg);
```

Comment:
Appreciate the spacing here - makes code easy to read :)

Reply:
So it looks like get_event_description is something we're going to want to reuse across multiple webpages, so instead of having the function copied, we should have it in a single spot that we import from.